src/style.css

```diff
@@ -0,0 +1,108 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 403px;
+  margin: 0;
+  padding: 30px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 15px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  margin: 0 0 20px;
+}
+
+.card--feedback .card__code {
+  margin: 0;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.card__item {
+  height: 15px;
+  width: 15px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.card--4 .card__item:nth-child(-n+4) {
```

Comment:
add classes from stars block task, so if I change the class of your element in dev tools I will see how the count of active stars changing.